code/main/core/configurationModule.brs

```diff
@@ -20,23 +20,35 @@ end function
 function _adb_ConfigurationModule() as object
     module = _adb_AdobeObject("com.adobe.module.configuration")
     module.Append({
-        CONFIG_KEY: AdobeAEPSDKConstants().CONFIGURATION,
+        _CONFIG_KEY: AdobeAEPSDKConstants().CONFIGURATION,
+        _MEDIA_CONFIG_KEY_PREFIX: "edgemedia.",
         _edge_configId: invalid,
         _edge_domain: invalid,
-        ' example config = {edge.configId:"1234567890", edge.domain:"xyz.net"}
+        ' example config = {edge.configId:"1234567890", edge.domain:"xyz.net"
+        _media_configuration: {},
```

Comment:
I think we can still keep the constants individual same as edge configuration. We can drop the prefix checks and try to read all configs. 
```
_media_playerName:
_media_channel:
_media_appVersion:
```